docs/administration/configuration/configuring-user-quotas/index.md

```diff
@@ -0,0 +1,65 @@
+
+REANA automatically creates two quota resources for you:
+
+- `cpu` to limit the amount of CPU time available to run workflows;
+- `disk` to limit the storage space available for file uploads.
+
+In addition to enabling quotas, you might want to decide how frequently quotas should be updated.
+This is done using two options: [`quota.periodic_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) and [`quota.workflow_termination_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana).
+
+The [`quota.periodic_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value defines when a periodic cron job should be scheduled to update quotas. This is specified using a [cron expression](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) and by default it is set to `"0 3 * * *"`, which means that it is scheduled to run once every night at 3AM.
+
+The [`quota.workflow_termination_update_policy`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value is responsible for defining which resources should be updated after the workflow is in its final state (finished, failed, stopped, deleted). By default, quotas are not updated at the end of the execution of a workflow.
+
+Below you can find an example configuration:
+
+```yaml
+quota:
+  enabled: true
+  periodic_update_policy: "0 3 * * *"  # everyday at 3AM
+  workflow_termination_update_policy: "cpu,disk"  # after workflow finishes or fails update its cpu and disk resources usage
+```
+
+## Setting default quota limits
+
+When user quotas are enabled, no limit on the usage of resources is enforced by default. You can set a default quota limit that will apply to all new users with the [`quota.default_disk_limit`](https://github.com/reanahub/reana/tree/master/helm/reana) and [`quota.default_cpu_limit`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm values:
+
+```yaml
+quota:
+  default_disk_limit: 5000000000  # bytes (4768 MiB) 
+  default_cpu_limit: 3600000      # milliseconds (1 hour)
+  ...
+```
+
+!!! warning
+    If the default quota values are changed, they will only be applied to newly created users. Users that already exist will not be affected by this change.
+
+## Setting quotas for selected users
+
+In certain situations, you might want to set different quotas for specific users.
+This can be done via the `reana-admin` tool that is present in the `reana-server` pod.
+
+Log in to the `reana-server` pod:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -- /bin/bash
+```
+
+You can now set a custom quota limit to selected users:
+
+```console
+# flask reana-admin quota-set -e john.doe@example.org -r disk -l 250000
+Quota limit 250000 for 'disk (shared storage)' successfully set to users ('john.doe@example.org',).
+```
+
+You can learn more about this command by running `flask reana-admin quota-set --help`.
```

Comment:
... learn more about the `quota-set` administrative command options by running...